jazzy/distribution.yaml

```diff
@@ -1106,6 +1106,10 @@ repositories:
       version: jazzy
     status: developed
   create3_sim:
+    doc:
+      type: git
+      url: https://github.com/iRobotEducation/create3_sim.git
+      version: jazzy
```

Comment:
This is a different branch than the source branch below.  Is that what you intend?

Reply:
Good catch.
the problem here is actually the source below, that should point to the `jazzy` branch